.github/workflows/ci.yml

```
@@ -119,6 +119,12 @@ jobs:
             gcr.io/iree-oss/base@sha256:5d43683c6b50aebe1fca6c85f2012f3b0fa153bf4dd268e8767b619b1891423a \
             ./build_tools/cmake/build_all.sh \
             "${BUILD_DIR}"
+      # The archive step below doesn't include these files. Remove them first to
+      # save disk space.
+      - name: "Removing unused files"
+        run: |
+          find "${BUILD_DIR}" -type f -name "*.a" -o -type f -name "*.o" \
+            -exec rm -v {} +
```

Can also just use `-delete` I think?

```suggestion
          find "${BUILD_DIR}" -type f -name "*.a" -o -type f -name "*.o" -delete
```